libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -133,7 +135,24 @@ public async Task ContinueConversationAsync(ConversationReference reference, Bot
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot bot, CancellationToken cancellationToken)
         {
-            if (!_facebookClient.VerifySignature(request))
+            if (request.Query["hub.mode"] == "subscribe")
```

Comment:
Could we use an Enum or some const varialbes here instead of having it harcoded?

Reply:
Done. Thanks!